<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>450</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#widget{
	border-image: url(:/background/background/ocean-3605547_1920.jpg);

color: rgb(255, 255, 255);
}
QLabel{
font: 25 10pt &quot;Microsoft YaHei&quot;;
border-radius: 4px;
background-color: argb(60, 60, 60, 135);
color: rgb(255, 255, 255);
}
Line{
	background-color: rgb(0, 85, 0);
}
</string>
    </property>
    <widget class="QLineEdit" name="lineEditCity">
     <property name="geometry">
      <rect>
       <x>575</x>
       <y>10</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(47, 47, 47, 130);
color: rgb(255,255, 255);</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelDate">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>3</y>
       <width>421</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>20</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 20pt &quot;黑体&quot;;
color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>2024-2-15</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="Line" name="hLine_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>45</y>
       <width>781</width>
       <height>1</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="hLine_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>183</y>
       <width>351</width>
       <height>1</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="hLine_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>335</y>
       <width>351</width>
       <height>1</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 20pt &quot;华文琥珀&quot;;
color: rgb(85, 255, 0);
background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>天气预报</string>
     </property>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>189</y>
       <width>353</width>
       <height>141</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="0">
       <widget class="QLabel" name="labelWindDirTitle">
        <property name="text">
         <string>风向</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelPM25Title">
        <property name="text">
         <string>PM2.5</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelWindForceTitle">
        <property name="text">
         <string>风力</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labelHumidityTitle">
        <property name="text">
         <string>湿度</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QLabel" name="labelWindDirection">
        <property name="text">
         <string>西北风</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QLabel" name="labelWindForce">
        <property name="text">
         <string>&lt;3</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QLabel" name="labelPM25">
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLabel" name="labelHumidity">
        <property name="text">
         <string>95%</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>192</x>
       <y>50</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_14">
        <property name="styleSheet">
         <string notr="true">border-image: url(:/icon/icon/Location.png);
background-color: rgba(60, 60, 60, 0);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="labelCity">
        <property name="styleSheet">
         <string notr="true">font: 16pt &quot;Microsoft YaHei UI&quot;;
color: rgb(255, 255, 255);
background-color: rgba(60, 60, 60, 0);</string>
        </property>
        <property name="text">
         <string>长沙</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>50</y>
       <width>437</width>
       <height>391</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="QLabel" name="labelDay01">
        <property name="text">
         <string>01/01</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="6" column="3">
       <widget class="QLabel" name="labelHighTpt04">
        <property name="text">
         <string>10℃</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="12" column="3">
       <widget class="QLabel" name="labelLowTpt04">
        <property name="text">
         <string>-1℃</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QLabel" name="labelHighTpt03">
        <property name="text">
         <string>10℃</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="2" rowspan="2">
       <widget class="QLabel" name="labelTypeIcon03">
        <property name="styleSheet">
         <string notr="true">border-image: url(:/weather_icon/weather_icons/多云.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="labelDay02">
        <property name="text">
         <string>02/02</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="labelDay03">
        <property name="text">
         <string>03/03</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QLabel" name="labelType05">
        <property name="text">
         <string>阴</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QLabel" name="labelQuality05">
        <property name="styleSheet">
         <string notr="true">color: rgb(170, 0, 0);</string>
        </property>
        <property name="text">
         <string>重度污染</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="labelWeek05">
        <property name="text">
         <string>周六</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLabel" name="labelHighTpt02">
        <property name="text">
         <string>10℃</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="1" rowspan="2">
       <widget class="QLabel" name="labelTypeIcon02">
        <property name="styleSheet">
         <string notr="true">border-image: url(:/weather_icon/weather_icons/多云.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLabel" name="labelQuality04">
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 0, 0);</string>
        </property>
        <property name="text">
         <string>中度污染</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="0" rowspan="2">
       <widget class="QLabel" name="labelTypeIcon01">
        <property name="styleSheet">
         <string notr="true">border-image: url(:/weather_icon/weather_icons/多云.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QLabel" name="labelType04">
        <property name="text">
         <string>阴</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="labelType03">
        <property name="text">
         <string>阴</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QLabel" name="labelDay06">
        <property name="text">
         <string>06/06</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelType01">
        <property name="text">
         <string>阴</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="7" column="0" rowspan="3" colspan="6">
       <widget class="QLabel" name="labelTptCurve">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="12" column="4">
       <widget class="QLabel" name="labelLowTpt05">
        <property name="text">
         <string>-1℃</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLabel" name="labelWeek06">
        <property name="text">
         <string>周日</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="labelDay04">
        <property name="text">
         <string>04/04</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="12" column="1">
       <widget class="QLabel" name="labelLowTpt02">
        <property name="text">
         <string>-1℃</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QLabel" name="labelDay05">
        <property name="text">
         <string>05/05</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="QLabel" name="labelQuality06">
        <property name="text">
         <string>良</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="12" column="0">
       <widget class="QLabel" name="labelLowTpt01">
        <property name="text">
         <string>-1℃</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="5">
       <widget class="QLabel" name="labelType06">
        <property name="text">
         <string>阴</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="12" column="2">
       <widget class="QLabel" name="labelLowTpt03">
        <property name="text">
         <string>-1℃</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelQuality01">
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 170, 0);</string>
        </property>
        <property name="text">
         <string>优质</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="5" rowspan="2">
       <widget class="QLabel" name="labelTypeIcon06">
        <property name="styleSheet">
         <string notr="true">border-image: url(:/weather_icon/weather_icons/多云.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="labelWeek04">
        <property name="text">
         <string>周五</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="labelQuality02">
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 0);</string>
        </property>
        <property name="text">
         <string>良好</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="3" rowspan="2">
       <widget class="QLabel" name="labelTypeIcon04">
        <property name="styleSheet">
         <string notr="true">border-image: url(:/weather_icon/weather_icons/多云.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="labelType02">
        <property name="text">
         <string>阴</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labelWeek01">
        <property name="text">
         <string>昨天</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="6" column="5">
       <widget class="QLabel" name="labelHighTpt06">
        <property name="text">
         <string>10℃</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="4" rowspan="2">
       <widget class="QLabel" name="labelTypeIcon05">
        <property name="styleSheet">
         <string notr="true">border-image: url(:/weather_icon/weather_icons/多云.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="labelHighTpt01">
        <property name="text">
         <string>10℃</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="6" column="4">
       <widget class="QLabel" name="labelHighTpt05">
        <property name="text">
         <string>10℃</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="labelQuality03">
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 170, 0);</string>
        </property>
        <property name="text">
         <string>轻度污染</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="labelWeek03">
        <property name="text">
         <string>明天</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="labelWeek02">
        <property name="text">
         <string>今天</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="12" column="5">
       <widget class="QLabel" name="labelLowTpt06">
        <property name="text">
         <string>-1℃</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="labelLeaf">
     <property name="geometry">
      <rect>
       <x>18</x>
       <y>58</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/icon/icon/leafs.png);
color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="labelQuality">
     <property name="geometry">
      <rect>
       <x>38</x>
       <y>58</y>
       <width>56</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>优</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="labelTemperature">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>241</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 40pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>9℃</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="labelType">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>120</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(60, 60, 60, 0);</string>
     </property>
     <property name="text">
      <string>晴</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="labelTpeIcon">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>63</y>
       <width>101</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(60, 60, 60, 0);
border-image: url(:/weather_icon/weather_icons/晴.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="labelTip">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>351</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(60, 60, 60, 0);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>天气提示：</string>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>205</x>
       <y>340</y>
       <width>1</width>
       <height>101</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>340</y>
       <width>191</width>
       <height>111</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(60, 60, 60, 130);
border_radius:3px</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;感&lt;br/&gt;冒&lt;br/&gt;指&lt;br/&gt;数&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QTextBrowser" name="txtBrowserGetCold"/>
      </item>
     </layout>
    </widget>
    <widget class="QToolButton" name="btnSearch">
     <property name="geometry">
      <rect>
       <x>722</x>
       <y>15</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/icon/icon/search.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QToolButton" name="btnRefresh">
     <property name="geometry">
      <rect>
       <x>753</x>
       <y>10</y>
       <width>37</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/icon/icon/refresh.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="labelSunRiseSet">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>350</y>
       <width>151</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>labelDate</zorder>
    <zorder>hLine_1</zorder>
    <zorder>hLine_2</zorder>
    <zorder>hLine_3</zorder>
    <zorder>label</zorder>
    <zorder>gridLayoutWidget</zorder>
    <zorder>gridLayoutWidget_3</zorder>
    <zorder>gridLayoutWidget_2</zorder>
    <zorder>labelLeaf</zorder>
    <zorder>labelQuality</zorder>
    <zorder>labelTemperature</zorder>
    <zorder>labelType</zorder>
    <zorder>labelTpeIcon</zorder>
    <zorder>labelTip</zorder>
    <zorder>line</zorder>
    <zorder>gridLayoutWidget_4</zorder>
    <zorder>btnRefresh</zorder>
    <zorder>lineEditCity</zorder>
    <zorder>btnSearch</zorder>
    <zorder>labelSunRiseSet</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
